content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+Documentation
+#############
+
+:menu_order: 007
+
+.. contents::
+
+
+General
+=======
+
+**What is documented?**
+
+**For the Chevah project members:**
+
+* Packages, modules, classes and functions using docstrings. 
```

Comment:
I think it's both what and how...
But I also saw this section more as an overview / summary of the main points in the page.
Will make a change for this to see if the next commit makes sense.